src/util/coordinate_calculation.cpp

```diff
@@ -322,7 +396,8 @@ double degreeToPixel(FloatLatitude lat, unsigned zoom)
 }
 
 // Converts a WMS tile coordinate (z,x,y) into a wsg84 bounding box
-void xyzToWSG84(const int x, const int y, const int z, double &minx, double &miny, double &maxx, double &maxy)
+void xyzToWSG84(
+    const int x, const int y, const int z, double &minx, double &miny, double &maxx, double &maxy)
 {
```

Comment:
Don't know whether it was formatted correctly prior. But I force the use of clang-format-3-6 now and I get this changes... no idea why.